library/std/src/sys/hermit/thread_local_dtor.rs

```diff
@@ -1,5 +1,6 @@
 #![cfg(target_thread_local)]
 #![unstable(feature = "thread_local_internals", issue = "none")]
+#![allow(unsafe_op_in_unsafe_fn)]
```

Comment:
Could we put these in `hermit/mod.rs` instead?

Reply:
> Could we put these in ``hermit/mod.rs`` instead?

I'm sorry, but I didn't understand what you said. What do "these" mean?